cmd/create.go

```diff
@@ -138,10 +141,21 @@ func create(cmd *cobra.Command, args []string) error {
 
 	if corr.Vars["corral_private_key"] == nil && corr.Vars["corral_public_key"] == nil {
 		logrus.Info("generating ssh keys")
-		privkey, _ := generatePrivateKey(2048)
-		pubkey, _ := generatePublicKey(&privkey.PublicKey)
-		corr.PrivateKey = string(encodePrivateKeyToPEM(privkey))
-		corr.PublicKey = string(pubkey)
+		if corr.Vars["corral_ssh_key_type"] == ED25519_KEY_TYPE {
```

Comment:
we should probably check these errors as well, even if we weren't before

Reply:
addressed